### PR TITLE
Implement wallet functions

### PR DESCRIPTION
**Pull request by hochiw**
## Features
- Added gas fee config
- Implemented sendToken, delegate, undelegate and collectAllRewards functions
- Introduced cosmos query client

based on #51 
refs #55

### CODE REVIEW

File: react-app/src/clients/queryClient.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export type ExtendedQueryClient = QueryClient &
+  AuthExtension &
+  BankExtension &
+  DistributionExtension &
+  StakingExtension &
+  GovExtension &
+  TxExtension;
+
+export const createQueryClient = async (
```
Comment (chihimng) [inline react-app/src/clients/queryClient.ts:27]: makeQueryClient?

**Review by chihimng — CHANGES_REQUESTED** [review]

File: react-app/src/api/cosmosAPI.ts [hunk 2]
```diff
@@ -102,6 +113,61 @@ export const useCosmos = (): ICosmosAPI => {
     [chainInfo.currency.coinMinimalDenom, signTx, getBalance, wallet]
   );
 
+  const signDelegateTokenTx = useCallback(
```
Comment (chihimng) [inline react-app/src/api/cosmosAPI.ts:119]: can we split these to own files, otherwise this file will grow very fast?

**Review by chihimng — CHANGES_REQUESTED** [review]

**Comment by chihimng** [comment]
plz rebase

File: react-app/src/api/cosmosAPI.ts (hunk 2 above)
Comment (hochiw) [reply react-app/src/api/cosmosAPI.ts:119]: shall we handle signing and broadcasting tx in `cosmosAPI.ts` and handle message making in a model file

**Comment by hochiw** [comment]
@chihimng updated :adore2x:

**Comment by chihimng** [comment]
@hochiw i pushed some commits to rename the make functions to newXXX 🙏

**Review by chihimng — APPROVED** [review]

**Pull request merged**